bot/role_management/role_management.py

```diff
@@ -131,6 +130,203 @@ async def module_role_error(self, ctx: commands.Context, error: commands.Command
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            message (discord.Message): The message from which the reaction roles should be removed.
+        """
+        if len(message.reactions) == 0:
+            await ctx.send(":x: Die angegebene Nachricht besitzt keine Reaction-Roles.")
+            return
+
+        if self._db_connector.check_reaction_role_uniqueness(message.id):
+            self._db_connector.remove_reaction_role_group(message.id)
+            log.info("A reaction role has been added to the message with id %s.", message.id)
+
+            await ctx.send(":white_check_mark: Die Reaction-Roles der angegebenen Nachricht sind nicht mehr "
+                           "\"exklusiv\".")
+        else:
+            self._db_connector.add_reaction_role_group(message.id)
+            await ctx.send(":white_check_mark: Die Reaction-Roles der angegebenen Nachricht sind nun \"exklusiv\".")
+
+    @add_reaction_role.error
+    @remove_reaction_role.error
+    @clear_reaction_roles.error
+    @toggle_reaction_roles_exclusiveness.error
+    async def reaction_role_error(self, _ctx: commands.Context, error: commands.CommandError):
+        """Error Handler for the `reactionrole` command group.
+
+        Handles specific exceptions which occur during the execution of this command. The global error handler will
+        still be called for every error thrown.
+
+        Args:
+            _ctx (discord.ext.commands.Context): The context in which the command was called.
+            error (commands.CommandError): The error raised during the execution of the command.
+        """
+        if isinstance(error, commands.BadArgument):
+            print("**__Error:__** Die von dir angegebene Nachricht/Rolle existiert nicht.")
+
+    @commands.Cog.listener(name='on_raw_reaction_add')
+    async def reaction_role_add(self, payload: discord.RawReactionActionEvent):
+        """Event listener which triggers if a reaction has been added by a user.
+
+        If the affected message is in the specified role channel and the added reaction represents one of the configured
+        reaction roles, the corresponding role specified in the db will be added to the user.
+
+        Args:
+            payload (discord.RawReactionActionEvent): The payload for the triggered event.
+        """
+        if payload.channel_id == self.ch_role.id and not payload.member.bot:
+            if self._db_connector.check_reaction_role_uniqueness(payload.message_id):
+                message = await self.ch_role.fetch_message(payload.message_id)
+
+                for reaction in message.reactions:
+                    if reaction.emoji != payload.emoji.name:
+                        await reaction.remove(payload.member)
+
+            role_id = self._db_connector.get_reaction_role(payload.message_id, payload.emoji.name)
```

Comment:
What happens if someone else adds a reaction to a message that does not have a role associated? 
Is this case handled?